content/en/tracing/setup_overview/configure_data_security/_index.md

```diff
@@ -13,12 +13,52 @@ aliases:
 ---
 ## Overview
 
-The performance data and traces that you're collecting with Datadog can contain sensitive information that you want to filter out, obfuscate, scrub, filter, modify, or just not collect.  Additionally, it may contain synthetic traffic that might cause your error counts to be inaccurate or Datadog to not accurately indicate the health of your services.
+The performance data and traces that you're collecting with Datadog can contain sensitive information that you want to filter out, obfuscate, scrub, filter, modify, or just not collect. Additionally, it may contain synthetic traffic that might cause your error counts to be inaccurate or Datadog to not accurately indicate the health of your services.
 
-The Datadog Agent and some tracing libraries have  options available to address these situations and modify or discard spans, and various options are described below.  These docs cover several common methods for configuring Tracer and Agent to achieve these security requirements.
+The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans, and various options are described below. These docs cover several common methods for configuring Tracer and Agent to achieve these security requirements.
 
 If your fine-tuning needs aren't covered and you need assistance, reach out to [the Datadog support team][1].
 
+## HTTP data collected
```

Comment:
I would move this section under the "Generalizing resource names and filtering baseline" section as one of the first questions are around PIIs and Ids in ressource names

Reply:
Done, see 93cbce69747bc2c4e5ad4fd55e60d6057cda81e1